module/clock/src/mod_clock.c

```diff
@@ -198,9 +198,10 @@ static int clock_bind(fwk_id_t id, unsigned int round)
     if (round == 1)
         return FWK_SUCCESS;
 
-    if (!fwk_id_is_type(id, FWK_ID_TYPE_ELEMENT))
+    if (!fwk_id_is_type(id, FWK_ID_TYPE_ELEMENT)) {
         /* Only element binding is supported */
         return FWK_SUCCESS;
+    }
```

Comment:
I am not sure we wanted to put braces here for since line ifs. I would wait for comment from others.
Etienne could you please specify if your checkpatch complains here?

Reply:
Actually I didn't see checkpatch complaining. This is more a personal feeling. Feel free to reject the change, yet I think it makes the code simpler.

Reply:
This change is alright by me - the current style is incorrect.

Reply:
ok, i keep these changes in the proposed series.